### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kytrinyx**
Link to the apply page that is integrated with our maintainer program management app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ The community is open to active open source maintainers on GitHub with a demonst
 
 _What is a maintainer?_ Maintainers are the stewards of an open source project. They spend much of their time thinking about the people and organizational aspects behind the project, beyond the project’s creative output (such as code). Maintainers might spend their time reviewing others’ contributions, responding to issues, evangelizing the project, or managing releases.
 
-We’re starting with a small, but growing, group of maintainers to ensure a good experience. If you’d like to join, [let us know](https://goo.gl/forms/0dgiaQXrmIV7OfsI3), and we’ll review it as we add more members.
+We’re starting with a small, but growing, group of maintainers to ensure a good experience. If you’d like to join, [let us know](maintainers.githubapp.com/apply), and we’ll review it as we add more members.
```
Comment (gr2m) [inline README.md]: I think you need to add the `https://` or it will be treated as a local link

**Review by gr2m — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kytrinyx) [inline README.md]: oops. Yes, thank you!

**Comment by kytrinyx** [comment]
Updated, thanks @gr2m

**Comment by MikeMcQuaid** [comment]
Nice work @kytrinyx!

**Review by hsoony159 — COMMENTED** [review]
README.md

File: README.md (hunk 1 above)
Comment (hsoony159) [reply README.md]: > I think you need to add the `https://` or it will be treated as a local link

**Pull request merged**